_includes/neurips/2020/about.neurips.html

```diff
@@ -0,0 +1,515 @@
+<div class="about-details" style="font-size: 18px">
+  <h2> LatinX in AI Research at NeurIPS 2020 </h2>
+  <section class="workshop-intro" id="workshop-intro" style="color: black">
+    <p>The LXAI Workshop at <a
+      href="https://neurips.cc/">NeurIPS</a> will be hosted virtually on Monday, December 7th, 2020.
```

Comment:
All external links should open in a new tab, please add target="_blank" to these link tags.